### DISCUSSION TITLE
HTML to PDF ?

### DISCUSSION THREAD

**Discussion by c0ze** [gettalong/hexapdf#275] [General]
Hello,
I'm looking for a solution to convert a simple html to PDF, is it possible to do with hexa ?
It is a simple jekyll resume html, if possible I would like to make it a plugin which outputs both html & pdf.
I have access to content, so I can plug it in before html creation (source is yaml) but converting directly from HTML would be much easier.

**Comment by gettalong** [comment]
There is nothing implemented in HexaPDF itself to convert HTML to PDF. That in itself would be a rather large undertaking because one would need to render HTML5 correctly which means supporting all HTML5 elements and CSS3.

Since you have access to the content, it would be best to transform the content to HTML and to PDF. This is what I do with websites where I need a PDF available from the content.
